docs/source/configuration/conditions.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: Conditions
+subtitle: Set conditions for when events or instruments are triggered
+description: 
+---
+
+Conditions set or restrict when an [instrument](./instruments) is mutated or an [event](./events) is triggered.
```

Comment:
That sentence doesn't make sense to me. It's the word "restrict". Should that be "restricted" or something else?

Reply:
rewording to just "Conditions set when an instrument ..."